### PR TITLE
Budget App : Ingodo

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Ingodo 💵
> In English "Wallet", Ingodo is a budget web application that helps to manage your budget, you will mainly have a list of transactions associated with their categories in order to keep on track of your spending.
> I build this project following [these instructions](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)

In this PR I : 
* Create a Ror app with Postgresql
* Setup the database with migrations
* Implement the MVC of the App
  *  Implement Sign up and login pages that :
    * Allow the user to register using a name, email, and password
    * Allow the user to log into the app using an email and password
    * Allow logged-in users to access categories and transactions pages
  * Implement a Home page (categories page) that : 
    * Allow the logged-in user to add a new category
    * Allow the logged-in user to access a list of its categories
  * Implement Transactions page
    * Allow logged-in user to see a list of transactions
    * Allow logged-in users to add a new transaction in a specific category
* Display correct data from the database
* Add style following [given design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
* Deploy the App on Heroku. [Live Demo](https://ingodo.herokuapp.com/)
* Make a video to present my project [Presentation Video](https://drive.google.com/file/d/1dDYqDeWQZ85lpwy5VLVoPn6FXsN0a2bq/view?usp=sharing)

### CODE REVIEW

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @ChrispaixK, 

Good job so far!
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Student-created units and integration tests. :+1: 
- All tests cases are running without errors :heavy_check_mark: 
- Your PR has a good summary and title. Keep up :rocket: 
- Your commit messages are short, descriptive, and meaningful :100: 
- Linters for this PR were setup correctly and working as expected :100: 



## Suggested changes 

- [ ] **Video Presentation:** Hi @ChrispaixK, great job :+1:  for taking that great amount of time to show forth the features of your project. As you know, it's very important to always adhere to the requirements of a project. The maximum length of the video was supposed to be 5 minutes but you extended it to 8~9 minutes. It seems you also forgot to highlight the piece of code you found interesting or proud of while building your project. Kindly make the necessary adjustments and move a step forward toward approval.

_Check the comments under the review._


## Optional Suggested 
N/A



You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app/models/transfer.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Transfer < ApplicationRecord
+  belongs_to :user, dependent: :destroy
```
[the remark this answers is not in the document]
Comment (ChrispaixK) [reply app/models/transfer.rb:2]: I don't think so, this comment is confusing if there is a way of doing this differently let me know.

File: app/views/transfers/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      <p>$<%= t.amount %></p>
+    </div>
+    </div>
+  <% end %>
+<% else %>
+<p class="txt-cntr">zero transfers for now</p>
+<% end %> 
+
+<a href="transfers/new">
+<button class="btn-submit">Add Transaction</button>
```
[the remark this answers is not in the document]
Comment (ChrispaixK) [reply app/views/transfers/index.html.erb:34]: I will , thank you

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**